sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1467,7 +1511,8 @@ def __init__(
       triggering_frequency=None,
       validate=True,
       temp_file_format=None,
-      ignore_insert_ids=False):
+      ignore_insert_ids=False,
+      with_auto_sharding=False):
```

Comment:
let's have a JIRA issue and a `TODO(BEAM-XXXX)` to track making this the default? : )

Reply:
Done.